platform-manageability-agent/Makefile

```
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+NAME ?= platform-manageability-agent
+GO_MOD ?= readonly
+BUILD_DIR ?= build/artifacts
+PMA_PACKAGE_BUILD_DIR ?= $(BUILD_DIR)/package
```

```suggestion
PACKAGE_BUILD_DIR ?= $(BUILD_DIR)/package
```
We should use the standard `PACKAGE_BUILD_DIR` variable for package builds